src/main/scala/is/hail/stats/package.scala

```diff
@@ -299,6 +302,31 @@ package object stats {
     result.toInt
   }
 
+  def binomTest(nSuccess: Int, n: Int, p: Double, alternative: String): Double = {
+    val kind = alternative match {
+      case "two.sided" => TestKind.TWO_SIDED
+      case "less" => TestKind.LOWER
+      case "greater" => TestKind.GREATER
+      case _ => fatal("didn't recognize option for alternative. Use one of [less, greater, two.sided]")
```

Comment:
s"Invalid alternative \"$alternative\". Must be \"two_sided\", \"less\" or \"greater\"."